From c87c4a2c9ec3082bc5314b28005f80fb56ff6971 Mon Sep 17 00:00:00 2001
From: Kuba Wojciechowski <nullbytepl@gmail.com>
Date: Thu, 1 Oct 2020 22:41:41 +0300
Subject: [PATCH] sepolicy: hack: include msmnile extra sepolicy

* fun fact: caf doesn't have a way to do that any less hacky way

Change-Id: Ic03495f968b196b9bb5c3604d49c89d2caf9d95b
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index 1dd7a3ec..ff698659 100644
--- a/Android.mk
+++ b/Android.mk
@@ -68,3 +68,5 @@ ifneq (,$(filter sdm845 sdm710, $(TARGET_BOARD_PLATFORM)))
     endif
 endif
 endif
+
+BOARD_SEPOLICY_DIRS += device/qcom/msmnile/sepolicy/vendor
-- 
2.17.1

